tools/python_tools/cuvslam_tools/bag2edex/make_mcap_metadata.py

```
@@ -0,0 +1,109 @@
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("mcap", help="Path to .mcap file")
+    parser.add_argument("--version", type=int, default=9)
+    args = parser.parse_args()
+
+    mcap_path = Path(args.mcap).resolve()
+    bag_dir = mcap_path.parent
+    metadata_path = bag_dir / "metadata.yaml"
+
+    result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)